week2/length.c

```diff
@@ -0,0 +1,80 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "length.h"
+
+//Function Definitions
+Input GetUserInput(char* input)
```

Comment:
should be const char* as best practice, otherwise you risk overwriting the buffer